coordinatorlib/src/test/scala/com/socrata/datacoordinator/truth/loader/sql/StupidSqlLoader.scala

```diff
@@ -1,14 +1,30 @@
-package com.socrata.datacoordinator
-package truth.loader
-package sql
+package com.socrata.datacoordinator.truth.loader.sql
 
 import java.sql.Connection
 
 import com.rojoma.simplearm.util._
-
-import com.socrata.datacoordinator.util.{RowIdProvider, RowVersionProvider}
+import com.socrata.datacoordinator._
 import com.socrata.datacoordinator.id.RowVersion
+import com.socrata.datacoordinator.truth.loader._
+import com.socrata.datacoordinator.util.{RowIdProvider, RowVersionProvider}
 
+/**
+ * For use in testing the real implementation, SqlLoader, of trait Loader
+ *
+ *    _____________________________________
+ *   / StupidSqlLoader should generate the \
+ *   | same sql as SqlLoader               |
+ *   |                                     |
+ *   | ... just in a more straightforward  |
+ *   \ (stupid) fashion                    /
+ *   -------------------------------------
+ *          \   ^__^
+ *           \  (oo)\_______
+ *              (__)\       )\/\
+ *                  ||----w |
+ *                  ||     ||
+ */
+// TODO: maybe we can think of a better way to test SqlLoader?
```

Comment:
"Make sure the clever fast thing acts the same as a simple slow thing" is a pretty standard way to test clever things...